master/setup.py

```diff
@@ -187,26 +187,31 @@ def make_release_tree(self, base_dir, files):
 except ImportError:
     pass
 else:
-    ## dependencies
+    # external dependencies
+    setup_args['dependency_links'] = [
+        'git+https://github.com/mcuadros/pynats@v0.1.0#egg=pynats-0.0.1',
```

Comment:
This is weird, but it is because the author of the package has inconsistent versions.